pkg/api/types.go

```diff
@@ -249,6 +249,10 @@ type Config struct {
 
 	// ImageWorkDir is the default working directory for the builder image.
 	ImageWorkDir string
+
+	// ImageScriptsPath is the path to the directory containing the assemble/run scripts located within the builder image.
+	// It should be specified as an absolute path (e.g. "/usr/libexec/s2i")
+	ImageScriptsPath string
```

Comment:
How about `ImageScriptsDir` to be more consistent with `ImageWorkDir` above

Also, `path` could imply either dir or file

Reply:
yeah that is better